docs/source/user-guide/concepts/pkg-search.rst

```diff
@@ -0,0 +1,90 @@
+=========================================
+Package search and install specifications
+=========================================
+
+Conda supports the following specifications for ``conda search`` and ``conda install``.
+
+Package search
+==============
+
+``conda search`` for a specific package or set of packages can be accomplished in several ways. This section includes information on the standard specification and the use of key-value pairs.
+
+Standard specification
+----------------------
+
+.. figure:: ../../img/conda_search.png
+    :width: 100%
+
+**channel**
+    (Optional) Can either be a channel name or URL. Channel names may include letters, numbers, dashes, and underscores.
+
+**subdir**
+    (Optional) A subdirectory of a channel. Many subdirs are used for architectures, but this is not required. Must have a channel and backslash preceeding it. For example: ``main/noarch``
+
+**name**
+    (Required) Package name. May include the ``*`` wildcard. For example, ``*py*`` returns all packages that have "py" in their names, such as "numpy", "pytorch", "python", etc.
+
+**version**
+    (Optional) Package version. May include the ``*`` wildcard or a version range(s) in single quotes. For example: ``numpy=1.17.*`` returns all numpy packages with a version containing "1.17." and ``numpy>1.17,<1.19.2`` returns all numpy packages with versions greater than 1.17 and less than 1.19.2.
```

Comment:
It would be good to mention how incomplete versions are treated. Is an ommitted level always treated the same as `*`?

E.g. is `numpy=1.17` exactly the same as `numpy=1.17.*`? Or are there differences? This is what #10985 was partially about and which I don't quite see fixed by this PR.

Are wildcards just a more explicit way of showing that a certain level is unconstrained?

Is it allowed to put a wildcard anywhere? Is this for example permitted `numpy=1.*.0`?